set-vars.md

```diff
@@ -0,0 +1,159 @@
+---
+title: SET (session settings)
+summary: The SET statement modifies the current settings for the client session.
+toc: false
+---
+
+The `SET` [statement](sql-statements.html) can modify one of the
+session setting variables.
+
+{{site.data.alerts.callout_danger}}In some cases, client drivers can drop and restart the connection to the server. When this happens, any session configurations made with <code>SET</code> statements are lost. It is therefore more reliable to configure the session in the client's connection string. For examples in different languages, see the <a href="build-an-app-with-cockroachdb.html">Build an App with CockroachDB</a> tutorials.{{site.data.alerts.end}}
+
+<div id="toc"></div>
+
+## Required Privileges
+
+No [privileges](privileges.html) are required to modify the session settings.
+
+## Synopsis
+
+General syntax:
```

Comment:
Is it possible to instead convert this into a railroad diagram like the other SQL statements?